dgraphql/resolvers/trxcache.go

```diff
@@ -0,0 +1,72 @@
+//Implement Least Recently Used cache to check for duplication transaction ID
+
+package resolvers
+
+import (
+	"container/list"
+)
+
+type TrxCache struct {
+	capacity int
+	data     map[string]interface{}
+	order    *list.List
```

Comment:
What I meant by `*string` is pointer to string.
The transaction ID strings are already stored in `data`, so I don't think `order` should be enough to store their pointers.

Reply:
Just use chatgpt to improve this one.
I convert data to `map[string]*list.Element`, so the key is transaction ID, and the value will be the pointer to `order` element.

This will help with a quick update for `order` as well as maintain the quick lookup for `data`.

Reply:
ChatGPT' s code is smarter, but I still don't think that `order` should be a pointer to `list` instance. Just being a list instance would be enough. I mean: 
```
order    list.List
```

Reply:
update to `list.List`